src/translations/screens/Profile.ts

```diff
@@ -439,6 +439,18 @@ export default orgSpecificTranslations(ProfileTexts, {
               'Standard billettkategori',
             ),
           },
+          travelToken: {
+            label: _(
+              'Bruk billett på reisekort / telefon',
+              'Use ticket on reisekort / telefon',
```

Comment:
Should we translate "reisekort"?

```suggestion
              'Use ticket on travel card / phone',
```

Reply:
Good catch. Fixed.